arangod/Cluster/RebootTracker.cpp

```diff
@@ -92,17 +91,17 @@ RebootTracker::RebootTracker(RebootTracker::SchedulerPointer scheduler)
 #endif
 }
 
-void RebootTracker::updateServerState(State state) {
+void RebootTracker::updateServerState(ServersKnown const& state) {
```

Comment:
move in 111 line doesn't work now.
You copy under lock.
Please fix it and make copy before lock